File: 1019.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
   
    int N;
    cin >> N;

    int horas = N / 3600;
    N %= 3600;
    int minutos = N / 60;
    N %= 60;
    int segundos = N;

    cout << horas << ":" << minutos << ":" << segundos << endl;

    return 0;
}

File: 1019.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) {
    return lines.shift();
};
var N = parseInt(prompt());
var horas = parseInt(N/3600);
N = N % 3600;
var minutos = parseInt(N/60);
N = N % 60;
var segundos = N;
console.log(horas+":"+minutos+":"+segundos);